reconcile/cli.py

```diff
@@ -3517,6 +3529,8 @@ def integrations_manager(
     image_tag_from_ref: dict[str, str] | None,
     upstream: str | None,
     image: str | None,
+    integrations_fluentd_image_tag: str | None,
+    integrations_fluentd_image: str | None,
```

Comment:
please reorder it to let `integrations_fluentd_image` before `integrations_fluentd_image_tag`, consistent with other usages

Reply:
Done

Reply:
please also update order of `@click.option`, usages in `run_integration` and `run`